### PR TITLE
feat(art-appraisals): Add page content

### PR DESCRIPTION
**Pull request by damassi**
The type of this PR is: **Feature**

This PR solves [GRO-1040]

### Description

This PR finishes up the Art Appraisals page and adds in remaining content. 

cc @artsy/grow-devs 

[GRO-1040]: https://artsyproduct.atlassian.net/browse/GRO-1040?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/v2/Apps/ArtAppraisals/ArtAppraisalsApp.tsx [hunk 1]
```diff
@@ -1,16 +1,176 @@
...
     </>
   )
 }
+
+const Header: React.FC = () => {
+  return (
+    <FullBleedHeader src="https://d7hftxdivxxvm.cloudfront.net?height=3197&quality=80&resize_to=fit&src=https%3A%2F%2Ffiles.artsy.net%2Fimages%2Fappraisals-header-image.png&width=2500">
+      <Flex
+        position="absolute"
+        top={-7}
```
Comment (damassi) [inline src/v2/Apps/ArtAppraisals/ArtAppraisalsApp.tsx:41]: @dzucconi - i could not figure out how to perfectly center this perfectly inside of the FullBleedHeader without this

Comment (dzucconi) [reply src/v2/Apps/ArtAppraisals/ArtAppraisalsApp.tsx:41]: I can help you with that tomorrow. Definitely not necessary :)

File: src/v2/Apps/ArtAppraisals/ArtAppraisalsApp.tsx [hunk 2]
```diff
@@ -1,16 +1,176 @@
...
+        <WhyUseArtsy />
+        <MeetOurSpecialists />
+      </Join>
     </>
   )
 }
+
+const Header: React.FC = () => {
+  return (
+    <FullBleedHeader src="https://d7hftxdivxxvm.cloudfront.net?height=3197&quality=80&resize_to=fit&src=https%3A%2F%2Ffiles.artsy.net%2Fimages%2Fappraisals-header-image.png&width=2500">
```
Comment (dzucconi) [inline src/v2/Apps/ArtAppraisals/ArtAppraisalsApp.tsx:38]: You don't want to pass this a pre-resized image. The component will do the sizing for you

File: src/v2/Apps/ArtAppraisals/ArtAppraisalsApp.tsx [hunk 3]
```diff
@@ -1,16 +1,176 @@
...
+const WhyUseArtsy: React.FC = () => {
+  return (
+    <GridColumns>
+      <Column span={6} pr={6}>
+        <ResponsiveBox aspectWidth={690} aspectHeight={720} maxWidth="100%">
+          <Image
+            src="https://d7hftxdivxxvm.cloudfront.net?height=2160&quality=80&resize_to=fit&src=https%3A%2F%2Ffiles.artsy.net%2Fimages%2Fappraisals-main-image.png&width=2070"
+            width="100%"
+            height="100%"
+            srcSet="https://d7hftxdivxxvm.cloudfront.net?height=2160&quality=80&resize_to=fit&src=https%3A%2F%2Ffiles.artsy.net%2Fimages%2Fappraisals-main-image.png&width=2070 1x, https://d7hftxdivxxvm.cloudfront.net?height=4320&quality=80&resize_to=fit&src=https%3A%2F%2Ffiles.artsy.net%2Fimages%2Fappraisals-main-image.png&width=4140 2x"
```
Comment (dzucconi) [inline src/v2/Apps/ArtAppraisals/ArtAppraisalsApp.tsx:85]: Either use the `resized` helper, like the `cropped` helper below; or neither.

File: src/v2/Apps/ArtAppraisals/ArtAppraisalsApp.tsx [hunk 4]
```diff
@@ -1,16 +1,176 @@
...
+            lazyLoad
+            alt=""
+          />
+        </ResponsiveBox>
+      </Column>
+      <Column span={6} mt={[2, 0]}>
```
Comment (dzucconi) [inline src/v2/Apps/ArtAppraisals/ArtAppraisalsApp.tsx:91]: Probably should be padding?

File: src/v2/Apps/ArtAppraisals/ArtAppraisalsApp.tsx (hunk 2 above)
Comment (damassi) [reply src/v2/Apps/ArtAppraisals/ArtAppraisalsApp.tsx]: I was wondering why there was no `srcSet`...

File: src/v2/Apps/ArtAppraisals/ArtAppraisalsApp.tsx (hunk 1 above)
Comment (damassi) [reply src/v2/Apps/ArtAppraisals/ArtAppraisalsApp.tsx:41]: Ty 👍 Loose my mind (and so much time, all the time) over stuff like this

**Pull request merged**